doc/source/examples/metrology/Creating Network Definitions for SOLT Calibration Standards.ipynb

```diff
@@ -0,0 +1,1116 @@
+      "text/plain": [
+       "<Figure size 432x288 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "mag = plt.subplot(1, 1, 1)\n",
+    "plt.title(\"Maury Microwave 8050CK10 Open (S11)\")\n",
+    "open_std.plot_s_db(color='red', label=\"Magnitude\")\n",
+    "plt.legend(bbox_to_anchor=(0.73, 1), loc='upper left', borderaxespad=0)\n",
+    "\n",
+    "phase = mag.twinx()\n",
+    "open_std.plot_s_deg(color='blue', label=\"Phase\")\n",
+    "plt.legend(bbox_to_anchor=(0.73, 0.9), loc='upper left', borderaxespad=0)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Short"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<matplotlib.legend.Legend at 0x72a2f675b4f0>"
+      ]
+     },
+     "execution_count": 17,
+     "metadata": {},
+     "output_type": "execute_result"
+    },
+    {
+     "data": {
```

Comment:
Results should be stripped from the notebook: https://scikit-rf.readthedocs.io/en/latest/contributing/index.html#contributing-to-the-documentation

Reply:
Fixed.